### PR TITLE
Bugfix/accessibility fixes

### PR DESCRIPTION
**Pull request by koops9**
Many fixes and refactoring on about semantic html and accessibility. Haven't fixed each and every problem but the base should be better than before in my view. Tried to also make the front end a bit clearer with changing some wordings.

### CODE REVIEW

File: frontend/src/components/BreadcrumbNav.jsx [hunk 1]
```diff
@@ -10,83 +10,115 @@ import {
...
-                className="TeamBreadCrumb"
-                id="TeamBreadCrumb"
-            >
-                {teamName}
-            </StyledLink>
-        </StyledInnerDiv>
+        <React.Fragment>
+            <BreadcrumbTeams />
+            <li>
+                <span>/</span>
```
Comment (tiiavalt) [inline frontend/src/components/BreadcrumbNav.jsx:28]: I think it would be wise to hide the "/"s from the screen reader.

File: frontend/src/components/BreadcrumbNav.jsx [hunk 2]
```diff
@@ -10,83 +10,115 @@ import {
...
+    build: <BreadcrumbItemBuild current />,
+    suite: <BreadcrumbItemSuite current />,
 };
 
 const BreadcrumbNav = ({ status }) => {
     return (
-        <BreadcrumbContainer id="breadCrumbNav" status={status}>
-            <StyledInnerDiv>{BREADCRUMB_STATUS[`${status}`]}</StyledInnerDiv>
+        <BreadcrumbContainer id={'breadCrumbNav'} status={status}>
+            <nav aria-label="Breadcrumb">
```
Comment (tiiavalt) [inline frontend/src/components/BreadcrumbNav.jsx:117]: I don't know what the policy is, but since Epimetheus has localization, it would be nice to use it also with aria-label.s

File: frontend/src/components/Footer.jsx [hunk 3]
```diff
@@ -7,18 +7,22 @@ const Footer = () => {
...
-                    <a href="https://www.siili.com">{t('footer.siili')}</a>
-                </span>
-                .
-            </TextStyles>
-        </FooterContainer>
+        <footer>
+            <FooterContainer>
+                <EpiIcon role="img" aria-label="Small Epimetheus logo">
+                    {t('footer.E')}
+                </EpiIcon>
```
Comment (tiiavalt) [inline frontend/src/components/Footer.jsx:14]: Is this really something that is needed to be seen/read by screen readers? In my opinion it's purely decorative.

File: frontend/src/components/MainNav.styles.js [hunk 4]
```diff
@@ -1,17 +1,19 @@
 ﻿import styled from 'styled-components';
 
-export const NavBar = styled.nav`
+export const NavBar = styled.div`
```
Comment (tiiavalt) [inline frontend/src/components/MainNav.styles.js:3]: I think `nav` was more accurate here than `div`. Why the change?

File: frontend/src/components/lastBuildElement/LastBuild.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ import BuildInfoTable from './BuildInfoTable';
 import { useParams } from 'react-router';
 import { useStateValue } from '../../contexts/state';
 
-const Divi = styled.div`
+const TableHeading = styled.div`
```
Comment (tiiavalt) [inline frontend/src/components/lastBuildElement/LastBuild.js:9]: Correct me if I'm wrong, but isn't this supposed to be `caption` instead of `div`?

File: frontend/src/components/team/Card.jsx [hunk 6]
```diff
@@ -1,25 +1,28 @@
 import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { CardSection, StyledSpan } from './Card.styles';
-import { H4 } from './SelectedTeam.styles';
+import { CardSection, StyledSpan, CardHeading } from './Card.styles';
+//import {Link} from 'react-router-dom';
```
Comment (tiiavalt) [inline frontend/src/components/team/Card.jsx:5]: This can probably be removed.

File: frontend/src/components/team/Card.jsx [hunk 7]
```diff
@@ -1,25 +1,28 @@
...
 
 const Card = ({ team, numberOfSeries }) => {
     const [t] = useTranslation(['team']);
     let history = useHistory();
-
     return (
-        <CardSection
+        <div
```
Comment (tiiavalt) [inline frontend/src/components/team/Card.jsx:11]: What's the reason for `div` instead of `a`? I think it's best practise to use semantic HTML always when possible.

File: frontend/src/components/team/SelectedTeam.jsx [hunk 8]
```diff
@@ -37,16 +37,18 @@ const TeamCard = ({ data }) => {
     const LastStarted = last_started.slice(0, 16);
     const testStatusIcon = pickIcon(last_status);
     return (
-        <CardSection>
-            <header>
-                <H4>{name}</H4>
+        <CardSection aria-label={name} tabIndex="0">
```
Comment (tiiavalt) [inline frontend/src/components/team/SelectedTeam.jsx:40]: `tabindex="0"` enables the user to navigate to the element and therefore it receives focus, but nothing else happens when clicking the element/pressing the return key. Instead, the screen reader navigates to the selected build.

**Review by tiiavalt — COMMENTED** [review]
Nice improvements!
Although, there was some regression at least with the footer. It used to be at the bottom of the page even though the main content's height didn't fully fill up the window's height. In this branch, the footer starts immediately when the main content ends: 
Screenshot 2021-02-17 at 12 50 29

This is probably something that can also be fixed later, but just letting you know!

File: frontend/src/components/MainNav.styles.js (hunk 4 above)
Comment (koops9) [reply frontend/src/components/MainNav.styles.js:3]: Lets delete this change

File: frontend/src/components/team/Card.jsx (hunk 6 above)
Comment (koops9) [reply frontend/src/components/team/Card.jsx:5]: Yes totally

File: frontend/src/components/team/SelectedTeam.jsx (hunk 8 above)
Comment (koops9) [reply frontend/src/components/team/SelectedTeam.jsx:40]: I didn't realixe the focus style applies also when clicking with mouse. I'm open for better solution here.

File: frontend/src/components/Footer.jsx (hunk 3 above)
Comment (koops9) [reply frontend/src/components/Footer.jsx]: I'll change this to aria-hidden

File: frontend/src/components/BreadcrumbNav.jsx (hunk 1 above)
Comment (koops9) [reply frontend/src/components/BreadcrumbNav.jsx:28]: Lets' add this to TODO list

File: frontend/src/components/BreadcrumbNav.jsx (hunk 2 above)
Comment (koops9) [reply frontend/src/components/BreadcrumbNav.jsx:117]: This will also have to be done in future with more knowledge about localization if that's be okay

**Comment by koops9** [comment]
[Accessibility fixes](https://trello.com/c/TIj47tbw/192-accessibility-fixes)

**Review by tiiavalt — APPROVED** [review]
